docs/path-resolution.rst

```diff
@@ -250,32 +250,23 @@ For example in ``./abc/..//`` there are three path segments: ``.``, ``abc`` and
 
 The compiler computes a source unit name from the import path in the following way:
 
-1. First a prefix is computed
-
-    - Prefix is initialized with the source unit name of the importing source unit.
-    - The last path segment with preceding slashes is removed from the prefix.
-    - Then, the leading part of the normalized import path, consisting only of ``/`` and ``.``
-      characters is considered.
-      For every ``..`` segment found in this part the last path segment with preceding slashes is
-      removed from the prefix.
-
-2. Then the prefix is prepended to the normalized import path.
-   If the prefix is non-empty, a single slash is inserted between it and the import path.
+#. The result is initialized with the source unit name of the importing source unit.
```

Comment:
What is the `result` is not very clear to me here. Why not keep it as prefix?

Reply:
Because the prefix is not the final result. It's just the part we start with. It made more sense when the final step that combines prefix with the import path was still here.

I changed it to "resolved name". Check if it's better now.

Reply:
Ah, I see. LGTM now :D